examples/oft_dreambooth/oft_dreambooth_inference.ipynb

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
The notebook assumes that the user ran the training script first to create an adapter, right? Let's mention that. Otherwise, we could maybe copy https://huggingface.co/takuoko/oft_test to our PEFT HF repo and link that one.

Reply:
I mentioned it now in [PR#2](https://github.com/okotaku/peft/pull/2). Don't know how to integrate the oft_test adapter into the PEFT HF repo so I would need some guidance to help with that.

Reply:
I think it's okay now as is, this way users who just open the notebook know what they should do first.